.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: 'Build Docker images'
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
```

Comment:
I think the `v` prefix is non-standard for Docker images (we did it wrong for `nuts-node`, I believe). I'd drop it.